app/helpers/GroupBindings/GroupBindingAccessor.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace App\Helpers\GroupBindings;
+
+use App\Model\Entity\Group;
+
+/**
+ * Provides access to instances of IGroupBindingProvider
+ */
+class GroupBindingAccessor {
+  /** @var IGroupBindingProvider[] */
+  private $providers = [];
+
+  public function addProvider(IGroupBindingProvider $provider) {
+    $this->providers[] = $provider;
+  }
+
+  public function getBindingsForGroup(Group $group) {
+    $result = [];
+    foreach ($this->providers as $provider) {
+      $result[$provider->getGroupBindingIdentifier()] = $provider->findGroupBindings($group);
```

Comment:
This is not working correctly with serialization. The GroupBinding objects are serialized as empty objects, but we need their codes.